### PR TITLE
Added travis ci support for modern node versions

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

**Comment by amilajack** [comment]
@h2non btw the tests are failing because of the linter, which is failing because of one error:

```bash
The command "npm run lint" exited with 1.
```

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
+  directories:
+    - node_modules
+
+install:
+  - npm install
+
+script:
+  - npm run test
+  - npm run lint
+  - npm run build
```
Comment (h2non) [inline .travis.yml]: This two commands are not supported in `package.json`. Please, only run `npm test`

**Comment by amilajack** [comment]
@h2non sorry about that. Fixed!

**Comment by tomas-fp** [comment]
Thanks!

**Pull request merged**